packages/admin/admin/src/table/filterbar/FilterBar.tsx

```diff
@@ -1,46 +1,61 @@
-import { ComponentsOverrides, Theme } from "@mui/material";
-import { createStyles, WithStyles, withStyles } from "@mui/styles";
+import { ComponentsOverrides, css, styled, Theme, useThemeProps } from "@mui/material/styles";
+import { ThemedComponentBaseProps } from "helpers/ThemedComponentBaseProps";
 import * as React from "react";
 
 /**
  * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
  */
+
 export type FilterBarClassKey = "root" | "barWrapper";
 
 /**
  * @deprecated Use MUI X Data Grid in combination with `useDataGridRemote` instead.
  */
-export interface FilterBarProps {
+
+const Root = styled("div", {
+    name: "CometAdminFilterBar",
+    slot: "root",
+    overridesResolver(_, styles) {
+        return [styles.root];
+    },
+})(css`
+    & [class*="CometAdminFormFieldContainer-root"] {
```

Comment:
These classes for Comet Admin components no longer exist after the refactor. 
But I'm not sure, how else to do this right now...